Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Sorcerer trait adaptation

#### Changelog:
- Added sorcerer trait placeholders (currently do nothing) from ck2 version ( Added 8 traits - 3 good inheritable, 3 bad inheritable, 1 good noninheritable, 1 bad noninheritable )
- Spawned witches, debutantes and warriors can appear with this trait
- Added sorcerer trait to level 4 blood legacy perk
- Added sorcerer trait as requirement to strengthen_bloodline_decision
- Reincarnation also can give sorcerer traits

#### Dev changelog:
- added `this_is_magical_society_trigger` (not active) to increase value of characters with sorcerer traits in elections.


# How to test:

- Check all of the above
- Check that ruler transition (death screen) localization related to new traits works.
- Proofread localization
